app/controllers/articles_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
 	def create
 		@article = Article.new(article_params)
 		if @article.save
-			redirect_to new_article_path, :notice => "Article successfully added"
+			redirect_to article_path(@article), :notice => "Article successfully added"
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.